deploy/role.yaml

```diff
@@ -72,5 +72,6 @@ rules:
   - contrail
   resources:
   - '*'
+  - contrailcnis
```

Comment:
Is it necessary if we specify '*' above?

Reply:
it's autogenerated by operator-sdk

Reply:
Removed manually and run operator-sdk. Didn't regeneratted *